lib/tar.ts

```diff
@@ -0,0 +1,49 @@
+import { readFileSync } from "fs";
+import { extract, Extract } from "tar-stream";
```

Comment:
Do we really want to use both `tar` and `tar-stream`? We have two different extraction approaches now within the same library. Lets use one please. For example, using `tar` and iterating over the list of files instead of streaming is probably perfectly fine.

Reply:
The only library we use is `tar-stream`